### DISCUSSION TITLE
Bug? 'Permission Denied' error when trying to run a bun script

### DISCUSSION THREAD

**Discussion by Ncage1974** [oven-sh/bun#12529] [Q&A]
Here is the script in my package.config:

```"db:seed": "./src/db/seed.ts"```

When I try to run it:
```bun run db:seed```

this is the error I receive:
```$ ./src/db/seed.ts -v
/usr/bin/bash: line 1: ./src/db/seed.ts: Permission denied
error: script "db:seed" exited with code 126 
```
Whats weird if I try to run it manually (at the same location):

```bun run "./src/db/seed.ts```

It works fine. I have been googling the error & haven't found a good answer. Anyone have any ideas? I have deleted my `node_modules` to make sure that wasn't the issue. I even get the error when running the command with sudo.

I'm using bun `1.1.8` & running inside a devcontainer.

**Comment by truongan07** [comment]
try ?
`sudo bun run db:seed `

**Reply by Ncage1974** [reply]
Yep tried that and unfortunately that didn't work either (same error)

**Reply by truongan07** [reply]
try 777 package.config ?

**Reply by Ncage1974** [reply]
Tried that & unfortunately same thing. Tonight i decided to try on another machine to see if it could be a machine issue. So i tried it on fedora linux (running same debian container) & same error. I guess at least its consistent.

**Reply by truongan07** [reply]
try absolutely path ?

**Comment by alexeyten** [comment]
Show output of the command `ls -l ./src/db/seed.ts`

and first line of `./src/db/seed.ts`.
